src/Compilers/CSharp/Test/Emit2/Semantics/InlineArrayTests.cs

```diff
@@ -21056,11 +21048,7 @@ static System.Collections.Generic.IEnumerable<int> Test()
 }
 ";
             var comp = CreateCompilation(src + Buffer4Definition, targetFramework: TargetFramework.Net80, options: TestOptions.ReleaseDll);
-            comp.VerifyDiagnostics(
-                // (6,26): error CS8176: Iterators cannot have by-reference locals
-                //         foreach (ref int y in GetBuffer())
-                Diagnostic(ErrorCode.ERR_BadIteratorLocalType, "y").WithLocation(6, 26)
-                );
+            comp.VerifyEmitDiagnostics();
```

Comment:
>comp.VerifyEmitDiagnostics(); [](http://example.com/codeflow?start=12&length=29)

Consider executing the code (and not throwing from `GetBuffer()`), for either this test or `Foreach_InIterator_07`.